contrib/pg_tde/src/encryption/enc_aes.c

```diff
@@ -251,12 +251,12 @@ AesGcmDecrypt(const unsigned char *key, const unsigned char *iv, const unsigned
 	return true;
 }
 
-/* This function assumes that the out buffer is big enough: at least (blockNumber2 - blockNumber1) * 16 bytes
+/*
+ * This function assumes that the out buffer is big enough: at least (blockNumber2 - blockNumber1) * 16 bytes
  */
 void
-Aes128EncryptedZeroBlocks(void *ctxPtr, const unsigned char *key, const char *iv_prefix, uint64_t blockNumber1, uint64_t blockNumber2, unsigned char *out)
+AesCtrBlocks(void *ctxPtr, const unsigned char *key, const char *iv_prefix, uint64_t blockNumber1, uint64_t blockNumber2, unsigned char *out)
```

Comment:
I think the `zero` is still useful in this name, as this is not a generic Ctr function, it only generates an encrypted stream of zeroes.

Reply:
I personally do not look at it as generating a stream of zeroes, it generates a stream of sequential numbers which are encrypted with ECB. It also happens to be the same stream as it would be if you had encrypted a stream of zeros but that is a perspective I find less useful.

Reply:
That said I am fine with their naming.

Reply:
Fixed.